spec/migrations/20221107195522_move_existing_iso_datastore_records_spec.rb

```diff
@@ -0,0 +1,36 @@
+require_migration
+
+describe MoveExistingIsoDatastoreRecords do
+  let(:ext_management_system_stub) { migration_stub(:ExtManagementSystem) }
+  let(:datastore_stub) { migration_stub(:IsoDatastore) }
+  let(:storage_stub) { migration_stub(:Storage) }
+
+  migration_context :up do
+    it "transfers datastore records to storages" do
+      emss = %w[Ovirt Redhat].map do |vendor|
+        ext_management_system_stub.create!(:type => "ManageIQ::Providers::#{vendor}::InfraManager")
+      end
+
+      emss.map do |ems|
+        datastore_stub.create!(:ext_management_system => ems)
+      end
```

Comment:
No need for `.map` if you aren't using the return
```suggestion
      emss.each { |ems| datastore_stub.create!(:ext_management_system => ems) }
```